src/gui/sdlmain.cpp

```diff
@@ -3902,6 +3903,7 @@ int sdl_main(int argc, char *argv[])
 
 		//If command line includes --help or -h, print help message and exit.
 		if (control->cmdline->FindExist("--help") ||
+		    control->cmdline->FindExist("-help") ||
```

Comment:
I don't think this should be changed in this PR? Seems it would break for anyone relying on the `--help` behavior.

Reply:
@shermp I don't think it will break anything. Instead, it just allows both --help and -help to work the same as -h. Not strictly part of this PR, but it is a one-line usability change so I include it in this PR.

Reply:
Oops, sorry about that. For some reason I thought you were removing the option. Doh.

Sorry about that!